packages/stk/stk_ngp_test/cmake/Dependencies.cmake

```diff
@@ -1,8 +1,16 @@
-SET(LIB_REQUIRED_DEP_PACKAGES Gtest Kokkos KokkosCore KokkosContainers)
+IF(TPL_ENABLE_GTest)
+  set(MAYBE_GTEST_TPL GTest)
+  set(MAYBE_GTEST_PACKAGE)
+ELSE()
+  set(MAYBE_GTEST_TPL)
+  set(MAYBE_GTEST_PACKAGE Gtest)
+ENDIF()
+
+SET(LIB_REQUIRED_DEP_PACKAGES Kokkos KokkosCore KokkosContainers)
```

Comment:
Not sure if it's appropriate to take GTest out of the LIB_REQUIRED_DEP_PACKAGES.  Although stk_ngp_test is only used for testing, I'd be surprised if the library itself built without Gtest.

Reply:
Good catch @tasmith4! Not intentional, definitely a mistake on my part

Reply:
Thanks for fixing @ibaned!